site/content/blog/2022-10-01-zap-and-hacktoberfest/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "ZAP and Hacktoberfest"
+summary: >
+    ZAP is partcipating in Hacktoberfest 2022.
+images:
+- https://www.zaproxy.org/site/blog/2022-10-01-zap-and-hacktoberfest/images/Hfest-Badge-2-Color-Void@2x.png
```

Comment:
I dont know if the '@' will cause problems with the social image?
It may well be fine but might be worth changing just in case as its really hard to test this before publishing 😉 
And we have had similar problems before..

Reply:
Yep sure... I just simoly need to change the filename :)